formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -372,8 +372,8 @@ let myFun =
    let myFun = fun ((X(hello) | Y(hello)) [@onOrPattern]) => hello;
    */
 /* Bucklescript FFI item attributes */
-[@bs.val] external imul : (int, int) => int =
-  "Math.imul";
+[@bs.val]
+external imul : (int, int) => int = "Math.imul";
```

Comment:
This is exactly how I'd want it to break.
Next, imagine if we drop the `[ ]` for "simple enough" attributes:

```reason
@bs.val
external imul : (int, int) => int = "Math.imul";
```